### ISSUE TITLE
2.4.0-beta1 release plan

### ISSUE THREAD

**Issue by joeparsons** [az-digital/az_quickstart#1704]
# Minor Release

## 2.4.0-beta1
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2.  Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

#### New feature
- #1608
- #1615
- #1642
- #1650
- #1681

#### Bug fix
- #1653

#### Breaking or visual change to existing behavior
- #1655
- #1628

#### Drupal core minor level update
- #1573

#### Drupal contrib project updates
- #1652
- #1610

#### Bug fix
- #1629 
- #1635
- #1624
- #1633
- #1625
- #1644
- #1692
- #1682 
- #1673

#### Drupal core - Patch level release
- #1661
- #1691

#### Drupal contrib - Patch level release
- #1618
- #1545

#### Other
- #1488
- #1696

**Comment by joeparsons** [comment]
# Release notes snippet
```
_This is a pre-release for the next minor version (feature release) of Arizona Quickstart 2.  Pre-releases are not supported for production sites, but they are intended for widespread testing in preparation for the upcoming stable release._

_This is the first Quickstart release using Drupal 9.4.x. No significant issues were encountered upgrading Quickstart from Drupal 9.3.x to 9.4.x._

_Please report bugs or problems with this pre-release to [az-digital@web.arizona.edu](mailto:az-digital@web.arizona.edu) or on [Slack](https://uarizona.slack.com/archives/C0SAW6H4J)._

## New changes (not included in 2.3.x)

#### New feature
- Closes #1607, Closes #1678: Change disable_login_form functionality, disable login form by default, add link to webauth to login form. (#1681)
- Closes #1623 - Add Split Screen component (#1650)
- Fixes #1626 Add calendar category filter (#1642)
- Closes #1611: Refine access for Quickstart 2 admin toolbar links. (#1615)
- Closes #1118 - Add Data Studio dashboard link to admin toolbar. (#1608)

#### Bug fix
- Closes #1513: Changes az_barrio style library setting to theme. (#1653)

#### Breaking or visual change to existing behavior
- Closes #1328: Truncate long emails on card displays of AZ Person (#1628)
- Closes #746: Adds bottom margin to Person, News, Events (#1655)

#### Drupal core minor level update
- Closes #1566: Prepare Quickstart for Drupal 9.4.x. (#1573)

#### Drupal contrib project updates
- Update drupal/media_library_edit requirement from 2.2.0 to 2.3.0 (#1610)
- Update drupal/chosen requirement from 3.0.2 to 3.0.3 (#1652)

## Backported changes (also to be included in 2.3.x)

#### Bug fix
- Closes #1476, #1520, #1521 Migrate az_remote_video (#1673)
- Closes #1557: Removed id hash from CSS selector for Search submit button (#1682)
- Closes #1489: Fix media permission denied for authenticated user role. (#1692)
- Closes #1643 Ensure Migrate Magician is enabled (#1644)
- Closes #1071, #1621: Use noConflict() method to fix Bootstrap/jQuery UI conflict breaking dialog close icons. (#1625)
- Closes #1630: Change demo content email addresses to @arizona.edu (#1633)
- Closes #1443 - Add bg-white to accordion's card-body (#1624)
- Closes #1634: Fix invalid count comparisons in existing hook_update_N() implementations. (#1635)
- Closes #1627 Skip az_paragraphs update if there are no paragraphs. (#1629)

### Drupal core - Patch level release
- Closes #1690: Update Drupal core to 9.3.17. (#1691)
- Security Issue #1660 Update Drupal core to 9.3.16 for SA-CORE-2022-011 (#1661)

#### Drupal contrib - Patch level release
- Update drupal/honeypot requirement from 2.1.0 to 2.1.1 (#1618)
- Closes #1544: Add workaround and patch for Paragraphs 1.14 migrate plugin issues. (#1545)

#### Other
- Update Arizona Bootstrap to 2.0.15 (#1696)
- Closes #1486 Adds a source migration plugin that filters out old content (#1488)
```

**Comment by joeparsons** [comment]
Release created: https://github.com/az-digital/az_quickstart/releases/tag/2.4.0-beta1
